tests/snuba/api/endpoints/test_organization_events_mep.py

```diff
@@ -2584,6 +2585,57 @@ def test_device_class_filter(self):
         assert meta["fields"]["device.class"] == "string"
 
 
+class OrganizationEventsMetricsEnhancedPerformanceEndpointTestWithOnDemandMetrics(
+    MetricsEnhancedPerformanceTestCase
+):
+    viewname = "sentry-api-0-organization-events"
+
+    def setUp(self):
+        super().setUp()
+
+    def do_request(self, query):
+        self.login_as(user=self.user)
+        url = reverse(
+            self.viewname,
+            kwargs={"organization_slug": self.organization.slug},
+        )
+        with self.feature(
+            {
+                "organizations:on-demand-metrics-extraction": True,  # XXX: We have coupling
```

Comment:
We need to deal with this or the customers we have enabled with the other flag are not going to be using extracted data.

If I'm reading the code correctly, this hinders our ability to test all of our changes, see the following:
https://github.com/getsentry/sentry/blob/0d84be95c9f41fb84075b6b97dce3230c6bf3ace/src/sentry/api/endpoints/organization_events.py#L251-L254
https://github.com/getsentry/sentry/blob/0d84be95c9f41fb84075b6b97dce3230c6bf3ace/src/sentry/api/endpoints/organization_events_stats.py#L255-L256